sgt-back-end/server.js

```diff
@@ -0,0 +1,149 @@
+import pg from 'pg';
+import express from 'express';
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+const app = express();
+app.use(express.json());
+
+// GET STARTS HERE
+
+app.get('/api/grades', async (req, res) => {
+  try {
+    const sql = `
+    select *
+      from "grades"
+      `;
+    const result = await db.query(sql);
+    const allGrades = result.rows;
+    if (allGrades) {
+      res.status(200).json(allGrades);
+    } else {
```

Comment:
This actually shouldn't ever happen. If the grades table is empty then `allGrades` will be an empty array (`[]`). If the grades table doesn't exist, an exception will be thrown. So at this point, `allGrades` will always be defined (or the process will never get to this line of code).